web/public/js/controller/pages/contact/contact.ctrl.js

```diff
@@ -0,0 +1,9 @@
+auroraApp.controller('ContactCtrl', function($scope, $controller) {
+  angular.extend(this, $controller('LoadableMixin', {$scope: $scope}));
+  	//reserved for possible email and message fields.
```

Comment:
This controller doesn't need to extend LoadableMixin - this was just there as a placeholder, but it doesn't need make any asynchronous requests.  You can remove the following timeout block too.